documentation/api/definitions.md

```diff
@@ -97,6 +97,7 @@ Object describing the list of methods to capture parameters for.
 |---|---|---|
 | `methods` | [MethodDescription](#methoddescription)[] | Array of methods to capture parameters for. |
 | `useDebuggerDisplayAttribute` | bool | Determines if parameters should be formatted using their [`DebuggerDisplayAttribute`](https://learn.microsoft.com/dotnet/api/system.diagnostics.debuggerdisplayattribute) if available and supported. Expressions in attributes may consist of properties, fields, methods without parameters, or any combination of these. |
+| `captureLimit` | int | The number of times to capture parameters before stopping. Note that parameters may continue to be captured for a short amount of time after this limit is reached. |
```

Comment:
Could we also update the examples in https://github.com/dotnet/dotnet-monitor/blob/main/documentation/api/parameters.md so that we have one with the `captureLimit`?

Reply:
Something I just thought of that I don't think there's a solution for, but maybe we can try to get ahead of it in the docs - people might not realize that the duration seconds still defaults to 30s when they use the capture limit. I can imagine setting a limit of 10 and being really confused when it stops before that, not realizing the duration has to be explicitly set to something higher (or indefinite) since it's whichever finishes first.

Reply:
Good point. Perhaps for the description we say something like:
` The number of times to capture parameters before stopping. If the specified duration elapses the operation will stop even if the capture limit is not yet reached`